# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: target/test/pkg_embutils.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We install embutils as system tools, but like to use the T2 default bash
which wrapper, ...

  - Rene Rebe <rene@exactcode.de>

--- embutils-0.19/Makefile.vanilla	2010-07-19 16:09:52.249046394 +0000
+++ embutils-0.19/Makefile	2010-07-19 16:09:57.186726111 +0000
@@ -11,7 +11,7 @@
 
 PRGS=sync echo chroot mkdir rmdir mknod pwd true false clear chown chgrp \
 chmod sleep tty tar basename df tee uname arch hostname env time dirname \
-domainname id ln mv cp yes which cat rm wc ls whoami mkfifo head install \
+domainname id ln mv cp yes cat rm wc ls whoami mkfifo head install \
 sosrm soscp sosmv sosln soslns md5sum sleep2 allinone uniq tr mesg du \
 uuencode uudecode nohup nice cmp mktemp truncate test date \
 printenv chrootuid renice \
